eq-author/src/App/introduction/Design/IntroductionEditor/index.js

```diff
@@ -0,0 +1,209 @@
+import React from "react";
+import gql from "graphql-tag";
+import styled from "styled-components";
+import { flowRight } from "lodash/fp";
+import { propType } from "graphql-anywhere";
+import PropTypes from "prop-types";
+
+import withPropRenamed from "enhancers/withPropRenamed";
+import withChangeUpdate from "enhancers/withChangeUpdate";
+
+import RichTextEditor from "components/RichTextEditor";
+import withEntityEditor from "components/withEntityEditor";
+
+import { colors } from "constants/theme";
+
+import transformNestedFragments from "utils/transformNestedFragments";
+
+import LegalBasisField from "./LegalBasisField";
+import CollapsiblesEditor from "./CollapsiblesEditor";
+
+import withUpdateQuestionnaireIntroduction from "./withUpdateQuestionnaireIntroduction";
+
+const Section = styled.section`
+  &:not(:last-of-type) {
+    border-bottom: 1px solid #e0e0e0;
+    margin-bottom: 2em;
+  }
+`;
+
+const Padding = styled.div`
+  padding: 0 2em;
+`;
+
+const SectionTitle = styled.h2`
+  font-size: 1.1em;
+  font-weight: bold;
+  color: ${colors.text};
+  margin: 0 0 1em;
+`;
+
+const SectionDescription = styled.p`
+  margin: 0.1em 0 1em;
+  color: ${colors.textLight};
+`;
+
+const titleControls = {
+  emphasis: true,
+  piping: true,
+};
+
+const descriptionControls = {
+  bold: true,
+  emphasis: true,
+  piping: true,
+  list: true,
+};
+
+export const IntroductionEditor = ({ introduction, onChangeUpdate }) => {
+  const {
+    id,
+    collapsibles,
+    title,
+    description,
+    secondaryTitle,
+    secondaryDescription,
+    tertiaryTitle,
+    tertiaryDescription,
+    legalBasis,
+  } = introduction;
+
+  return (
+    <>
+      <Section>
+        <Padding>
+          <SectionTitle style={{ marginBottom: "0" }}>
+            Introduction content
+          </SectionTitle>
+          <SectionDescription>
+            This content is displayed above the “start survey” button. The title
+            is not editable.
+          </SectionDescription>
+          <RichTextEditor
+            id="intro-title"
+            name="title"
+            label="Title"
+            value={title}
+            controls={titleControls}
+            size="large"
+            disabled
+            onUpdate={onChangeUpdate}
+            testSelector="txt-intro-title"
+          />
+          <RichTextEditor
+            id="intro-description"
+            name="description"
+            label="Description"
+            value={description}
+            controls={descriptionControls}
```

Comment:
Looking at this seems to suggest piping is enabled on description but it seems to be disabled for me, not sure if I'm missing something?